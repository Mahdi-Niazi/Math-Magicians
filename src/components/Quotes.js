import Quote from '../pages/Quote';

const Quotes = () => <Quote />;

export default Quotes;
