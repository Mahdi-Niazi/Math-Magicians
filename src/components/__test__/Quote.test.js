import renderer from 'react-test-renderer';
import Quotes from '../Quotes';

it('Snapshot of the Quote page', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
