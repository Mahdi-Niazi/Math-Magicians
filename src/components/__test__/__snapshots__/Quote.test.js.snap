// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the Quote page 1`] = `
<div
  className="quotes"
>
  <p>
     Loading...
  </p>
</div>
`;
